code/lib/cli/src/generate.ts

```diff
@@ -142,31 +141,13 @@ program
   );
 
 program
-  .command('repro [outputDirectory]')
```

Comment:
@storybookjs/core How would we feel about dropping the `repro` name and replacing it with `sandbox` instead, everywhere?

`sb sandbox`
`github.com/storybookjs/sandboxes`
`Generate and push sandboxes to Github`
`yarn publish-sandboxes`

Reply:
Simplifying this language was discussed before, I'm not sure we had a final say?

Reply:
I'm not in storybook/core, but hopefully you don't mind if I share my opinion.  If the intent is not that users should be using this command to generate reproductions to post with their issues, then I agree that `sandbox` is a better command.  To me, it implies something that is done for development rather than issue posting.  We would probably need to do a pass through the docs to see where `sb repro` is mentioned.

I think we should still have an easy way for users to create reproductions, ideally in something like Stackblitz.  But we can figure that out another time.

Reply:
thanks for chiming in @IanVS and you're definitely core (at least in my eyes)! Thanks a lot for mentioning about the docs, I'm going through it.

I like the word `repro` because it implies creation of minimal reproduction projects, and it feels like an internal command rather than a user facing command. I'm curious about other's opinions too!

Reply:
As for Stackblitz @IanVS, you can find links to each repro in the README https://github.com/storybookjs/repro-sandboxes

Reply:
I think we should have a single word for `repro`, `template`, `sandbox`, `project`.

Though creating a reproduction is often the desired end goal, the command doesn't generate a repro, it generates a sandbox unto which someone makes changes to repro something.

I have a slight preference for `sandbox` because of this. Neither word implies internal or external facing to me.

Reply:
I agree with @ndelangen. All the terms mentioned mean the same thing to me mentally, but sandbox is more immediately understandable. If I found the `sb repro` command I would look up what it did, while `sb sandbox` would be clear to me what it did.

> I think we should have a single word for repro, template, sandbox, project.

Agree here. It just occured to me from this conversation that `repros` and `sandbox` are the same thing, while I've always thought they were different somehow.
You could argue it makes sense to differentiate between `sandbox` as instances and `sandboxTemplate | template` as templates.

Reply:
The sandbox step of the task runner essentially does two things: (1) calls `repro` and (2) links in all the stories from the monorepo.

So I was being pendantic I'd say a "sandbox" is different to a "repro":
 - a "repro" is an empty SB project from some particular "template".
 - a "sandbox" is a repro with a bunch of our test stories inside it.

But that's pretty inside baseball. Users don't care about that. Do we need to keep the names separate?

I do think it'd be confusing to have a `repros/` directory in the monorepo that you did development in, so we want to unify the names, I'd vote for "sandbox".

Reply:
So here's an official proposal:

command:
```
sb sandbox <<template-name>>
```

repo:
`github.com/storybookjs/sandboxes`
which contains `templates` to generate sandboxes.

sandboxes can be used by end-users to report reproduction. They
1. generate a sandbox
2. make modifications, _demonstrating a bug_
3. push the sandbox with changes to a repository
4. report this repository to maintainers. We can refer to this final repository as a repro

The command can use the word "template" when asking the user to make a choice

Reply:
I think it's ok! We can just think of that step as "sandbox and link stories", called sandbox for short

Reply:
I'm not confused by that at all. :)

Reply:
Alright, I'm making the change! This will affect a bunch of files, so let me know something off.
Thank you all!